unity/Assets/Maroon/scenes/experiments/ThrowingBalls/scripts/FileController.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Newtonsoft.Json;
+using System.IO;
+using Maroon.GlobalEntities;
+
+public class FileController : MonoBehaviour
+{
+    private static FileController _instance;
+    [SerializeField] private List<TextAsset> _jsonFile = new List<TextAsset>();
+    private List<Parameters> _parameters;
+
+    /// <summary>
+    /// Class for storing parameters from a JSON-File
+    /// </summary>
+    [System.Serializable]
+    public class Parameters
+    {
+        public string Background;
+        public string Particle;
+        public string FunctionX;
+        public string FunctionY;
+        public string FunctionZ;
+        public float Mass;
+        public float T0;
+        public float DeltaT;
+        public float Steps;
+        public float X;
+        public float Y;
+        public float Z;
+        public float Vx;
+        public float Vy;
+        public float Vz;
+
+    }
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        // Listener for extern json data 
+        WebGlReceiver.Instance.OnIncomingData.AddListener(data =>
+        {
+            LoadExternJsonFile(data);

```

Comment:
replace it with method group
e.g. WebGlReceiver.Instance.OnIncomingData.AddListener(LoadExternJsonFile);